.github/actions/storybook/action.yml

```diff
@@ -4,6 +4,9 @@ inputs:
   percy-token:
     description: 'Token for Percy'
     required: true
+  skip-percy:
+    description: 'Whether or not to skip Percy - if skipped build still runs and preview url is still generated'
+    required: false
```

Comment:
should we use default here?

Reply:
I just check for `== 'true'` below so it isn't needed, but yeah I can add

Reply:
Thanks for adding that. All LGTM.